### PR TITLE
Add Glossary page to website

### PR DESCRIPTION
**Pull request by KyeMaloy97**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Adds the new glossary page to the website, and update the development markdown guide to show how to add to the list. Also bakes in MDX support.

## Related Issues

#282

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -157,5 +157,30 @@ gatsby serve
...
+
+```
+>### Term Name
+This is where you would write a brief description explaining the term.  
+Feel free to include links, images, and inline code snippets. Code blocks are supported.
+```
+
+To have a new line work correctly, you will need to end the sentence with **two** spaces before pressing the enter key (this is quirk of Markdown).
+
+To have an empty line, you can insert an empty character which can be found with some minor Googling.
```
Comment (helenmasters) [inline DEVELOPMENT.md]: If we know people will need to do some "minor Googling" to find this why don't we tell them what it is, or point at an example?

File: content/docs/glossary.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Glossary
+---
+
+# Glossary
+A page designed to explain some of the new terms you may come across as you explore Appsody.
```
Comment (SueChaplain) [inline content/docs/glossary.md]: I'd just go for something like 

*Learn about terms and definitions that are related to the Appsody project.*

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -157,5 +157,30 @@ gatsby serve
...
+
+In order to render correctly, definitions need to be written in the following way:
+
+```
+>### Term Name
+This is where you would write a brief description explaining the term.  
+Feel free to include links, images, and inline code snippets. Code blocks are supported.
+```
+
+To have a new line work correctly, you will need to end the sentence with **two** spaces before pressing the enter key (this is quirk of Markdown).
```
Comment (SueChaplain) [inline DEVELOPMENT.md]: For a new line, end the sentence with **two** spaces before pressing the enter key.
For an empty line, insert an empty character. (Tell them what the char is)

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -157,5 +157,30 @@ gatsby serve
...
+
+To have an empty line, you can insert an empty character which can be found with some minor Googling.
+
+The large letters, marking the letter of the alphabet the definitions belong to, are simply written with the following syntax:
+
+```
+## Letter
+```
+
+> The glossary is not self-organising, so  try to add defnitions in their correct place alphabetically.
```
Comment (SueChaplain) [inline DEVELOPMENT.md]: defnitions / definitions ... looks like an extra space between "so" and "try"

File: DEVELOPMENT.md (hunk 1 above)
Comment (KyeMaloy97) [reply DEVELOPMENT.md]: The character they need to use is invisible, I thought about doing something like: (" ") and including the symbol in the quote marks? It's some strange Unicode character that is available at his site: http://emptycharacter.com/

File: DEVELOPMENT.md (hunk 3 above)
Comment (KyeMaloy97) [reply DEVELOPMENT.md]: The char they have to copy is from a website which has a copy to clipboard button. Pasting the character into the markdown shows nothing (it works but is almost like it doesnt exist).

**Comment by KyeMaloy97** [comment]
Opened a new PR with no MDX support here #450 so we can get this in without being blocked by the CL reference being broken (https://github.com/appsody/appsody/issues/761)

**Pull request closed without merging**